src/components/NavHeader/NavHeader.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+
+export default function NavHeader() {
+  return (
+    <div className='header'>
+      <ul>
+        <li>
```

Comment:
When writing Navlinks, we can place them within divs to make styling easier. The bullets with li's can become hard to style, just an FYI 👍